### PR TITLE
Allow automatic changes with strings

### LINKED ISSUE #186: Automatic selection between string distributions
Currently by default only the regex distributions are matched.

### PR DESCRIPTION
**Pull request by qubixes**
Fixes #186 

New behavior is:
- Average char count < 25 -> RegexModel
- Average char count >= 25 && Language found -> FreeText
- Average char count >= 25 && Language **not** found -> RegexModel

### CODE REVIEW

File: examples/getting_started.ipynb
Comment (Samuwhale) [inline examples/getting_started.ipynb:1]: good one!

**Review by Samuwhale — APPROVED** [review]
Looks good to me!

File: examples/getting_started.ipynb
Comment (qubixes) [reply examples/getting_started.ipynb:1]: Actually, I shouldn't have changed it. But I keep being annoyed at it (while understanding that it is useful, especially for the online notebook hosting..

**Review by vankesteren — APPROVED** [review]

**Comment by vankesteren** [comment]
merging so I can use this in my pilot today

**Pull request merged**